src/test/java/uk/ac/ebi/eva/pipeline/jobs/AnnotationJobTest.java

```diff
@@ -137,7 +138,7 @@ public void allAnnotationStepsShouldBeExecuted() throws Exception {
         int consequenceTypeCount = 0;
         while (cursor.hasNext()) {
             count++;
-            DBObject dbObject = (DBObject) cursor.next().get("annot");
+            DBObject dbObject = (DBObject) cursor.next().get(VariantToDBObjectConverter.ANNOTATION_FIELD);
```

Comment:
This is statically imported in some classes and non-statically in others. I don't have a particular preference for any of those forms, but it should be homogeneous.